### ISSUE TITLE
Print the branch name in logging (alone with the existing tag and commit)

### ISSUE THREAD

**Issue by billmeek** [MythTV-Clients/MythtvPlayerForAndroid#148]
@dmfrey would you consider adding this to the app somewhere, such
that pressing on the version (for example) would bring up a list containing
the branch, tag and commit?

**Comment by dmfrey** [comment]
we could just add an About item in the nav drawer that just pops a dialog.
If you want to provide me a layout of everything you want on it, I can add
it. The layout can just be a simple mock up in a text doc if you like.

**Comment by billmeek** [comment]
That would be great, Whatever is easiest. I was thinking along the lines of
an Easter Egg, not really documented, but available to testers/users who
build from source etc. Or maybe alone the lines of a more traditional Help->
About menu item.

Along the lines of this:
```
Branch: master
Tag: mythtv-player-2.0.3
Commit: 1234567
```
With the data coming from the existing: BuildConfig.APPLICATION_BRANCH,  BuildConfig.APPLICATION_TAG and BuildConfig.APPLICATION_SHA1

**Comment by billmeek** [comment]
Sorry, BuildConfig.APPLICATION_BRANCH only exists in the 
remotes/origin/quickfix/print_branch_in_log branch. I didn't do
a PR on it yet. Just saw your: feature/issue_148_about_dialog

**Comment by dmfrey** [comment]
branch may not work in this case. We will always be building out of a release/xxxx. If you are ok with that, I will leave it in.

**Comment by billmeek** [comment]
Branch works perfectly! Right now, I'm getting: feature/issue_148_about_dialog.

Tag is of interest too, but that's really just a double check. E.g. right now
the tag is mythtv-player-2.0.2 even though I know we're on 2.0.3. Actually,
I don't know why the tag is off. It's OK when I switch to branch master.

I'd prefer to see to see the commit too, but Branch is really the key for me
as I can tell which one I'm on during tests. In LogCat, I see:
```
LogCat
MainActivity: Branch: feature/issue_148_about_dialog, Tag: mythtv-player-2.0.2, Commit: 5ebcf6f
```

**Comment by dmfrey** [comment]
Great, will merge this into develop.

**Comment by dmfrey** [comment]
closes #148
